167. Two Sum II - Input array is sorted.java

```diff
@@ -0,0 +1,16 @@
+public class Solution {
+    public int[] twoSum(int[] numbers, int target) {
+        int[] output = new int[2];
+        for (int i = 0; i < numbers.length - 1; i++) {
```

Comment:
Can you think of an algorithm that uses the fact that numbers is already sorted?